src/gui/properties/propertieswidget.ui

```diff
@@ -871,6 +871,22 @@
                </widget>
               </item>
               <item row="4" column="0">
+               <widget class="QLabel" name="labelPrivate">
```

Comment:
IMO it should be placed above "Info Hash v1".
Like so:
```
...
Added On: | Completed On: | Created On:
Private:
Info Hash v1:
...
```

Reply:
Maybe better 
```
...
Added On: | Completed On: | Created On:
Info Hash v1: | Private:
...
```
it'll save space.

Reply:
Personally, I prefer @Chocobo1 order. If space is the issue, I would find it more logical to put the two hashes on to the same line, rather than Hash v1 and Private.

Reply:
@ManiMatter I would still prefer to keep the hash to two lines.
@thalieht @glassez do you have your own preferences?

Reply:
@stalkerok - I moved the private flag up as per @Chocobo1 and left the hashes on two lines for now. 
Screenshots of current implementation: https://github.com/qbittorrent/qBittorrent/pull/20951#issuecomment-2184175736

Reply:
I don't have an opinion on this.

Reply:
@ManiMatter, however, I would still put it on one line. Or so, if possible, then the hash would definitely not overlap
```
...
Added On: | Completed On: | Created On:
Info Hash v1: | <empty> | Private:
...
```

Reply:
Anyway, it's not that important, maybe some more information will take up the empty space in the future.